doc/nrf/glossary.rst

```diff
@@ -455,7 +491,7 @@ Glossary
       A timer that causes a system reset if it is not poked periodically.
 
    West
-      A multiple repository management system used by Zephyr and the |NCS|.
+      A management system for multiple repositories used by Zephyr and the |NCS|.
```

Comment:
Why not use the standard definition in Zephyr's documentation:

The Zephyr project includes a swiss-army knife command line tool named west. West is developed in its own repository.
West’s built-in commands provide a multiple repository management system with features inspired by Google’s Repo tool and Git submodules. West is also “pluggable”: you can write your own west extension commands which add additional features to west. Zephyr uses this to provide conveniences for building applications, flashing and debugging them, and more.

Reply:
Something like this would sound better?

A command-line tool providing a management system for multiple repositories, used by Zephyr and the |NCS|.
It is expandable, as you can write your own extension commands to add additional features.
It is developed in its own repository (link).

Reply:
Sounds good @FrancescoSer

Reply:
fixed